### PR TITLE
Refactored services using utils and BusinessConverter

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/fop.py [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
+from business_register.converter.business_converter import BusinessConverter
 from data_ocean.converter import BulkCreateUpdateManager, Converter
-from data_ocean.models import Authority, Status, TaxpayerType
+from data_ocean.models import Register
+from data_ocean.utils import get_first_word, cut_first_word, format_date_to_yymmdd
 
 
-class FopConverter(Converter):
-    LOCAL_FILE_NAME = "fop.xml"
+class FopConverter(Converter, BusinessConverter):
```
Comment (roman-tiukh) [inline business_register/converter/fop.py:11]: 2 parent classes? are you really need this? Bad practice.  Valid only with Mixins

File: business_register/converter/fop.py [hunk 2]
```diff
@@ -35,23 +31,23 @@ def update_fop_fields(self, hash_code, status, registration_date, registration_i
...
+            status=status,
+            registration_date=registration_date,
+            registration_info=registration_info,
+            estate_manager=estate_manager,
+            termination_date=termination_date,
+            terminated_info=terminated_info,
+            termination_cancel_info=termination_cancel_info,
+            contact_info=contact_info,
+            vp_dates=vp_dates,
+            authority=authority
```
Comment (roman-tiukh) [inline business_register/converter/fop.py:50]: ?????????

File: data_converter/settings_local.base.py [hunk 3]
```diff
@@ -1,52 +0,0 @@
...
-CSRF_TRUSTED_ORIGINS = [
-    'dataocean.us',
-    'localhost',
-    '127.0.0.1',
-]
-
-LOCAL_FILE_NAME_KOATUU = ''
-LOCAL_FOLDER = ''
-FILE_URL_KVED = ''
-LOCAL_FILE_NAME_KVED = ''
```
Comment (roman-tiukh) [inline data_converter/settings_local.base.py:52]: ???
are you accidentally add this deletions?
look carefully what you commit and push.

File: location_register/converter/drv.py [hunk 4]
```diff
@@ -5,35 +5,27 @@
...
 from data_ocean.converter import Converter
+from business_register.converter.business_converter import BusinessConverter
 from location_register.models.drv_models import (DrvAto, DrvBuilding,
                                                  DrvCouncil, DrvDistrict,
                                                  DrvRegion, DrvStreet, ZipCode)
+from data_ocean.utils import clean_name, change_to_full_name
 
 
-class DrvConverter(Converter):
+class DrvConverter(Converter, BusinessConverter):
```
Comment (roman-tiukh) [inline location_register/converter/drv.py:15]: same as prev comment

File: location_register/converter/drv.py [hunk 5]
```diff
@@ -77,14 +68,13 @@ def parse_atos_data(self, region):
         return(response_to_dict['ATO'])
```
Comment (roman-tiukh) [inline location_register/converter/drv.py:68]: redundant braces
```suggestion
        return response_to_dict['ATO']
```

File: business_register/converter/kved.py [hunk 6]
```diff
@@ -3,7 +3,6 @@
 from data_ocean.models import Register
 
 class KvedConverter(Converter):
-    LOCAL_FILE_NAME = "kved.json"
     API_ADDRESS_FOR_DATASET = Register.objects.get(source_register_id="e1afb81c-70e4-4009-96a0-b240c36e4603").api_address
```
Comment (roman-tiukh) [inline business_register/converter/kved.py:6]: move this to `__init__`

File: business_register/converter/fop.py [hunk 7]
```diff
@@ -1,27 +1,23 @@
...
 from data_ocean.converter import BulkCreateUpdateManager, Converter
-from data_ocean.models import Authority, Status, TaxpayerType
+from data_ocean.models import Register
+from data_ocean.utils import get_first_word, cut_first_word, format_date_to_yymmdd
 
 
-class FopConverter(Converter):
-    LOCAL_FILE_NAME = "fop.xml"
+class FopConverter(Converter, BusinessConverter):
     API_ADDRESS_FOR_DATASET = Register.objects.get(source_register_id="1c7f3815-3259-45e0-bdf1-64dca07ddc10").api_address
```
Comment (roman-tiukh) [inline business_register/converter/fop.py:12]: move this line to `__init__`

File: business_register/converter/fop.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/converter/fop.py:50]: that`s the rule when you initialize an object

File: data_converter/settings_local.base.py (hunk 3 above)
Comment (IegorSoboliev) [reply data_converter/settings_local.base.py:52]: We need this file?

Comment (roman-tiukh) [reply data_converter/settings_local.base.py:52]: Yes. This a example for settings_local.py...

File: business_register/converter/fop.py (hunk 2 above)
Comment (roman-tiukh) [reply business_register/converter/fop.py:50]: ohh, i missed this) sorry

File: business_register/converter/fop.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/fop.py:11]: Fixed

File: data_converter/settings_local.base.py (hunk 3 above)
Comment (IegorSoboliev) [reply data_converter/settings_local.base.py:52]: Restored

File: location_register/converter/drv.py (hunk 4 above)
Comment (IegorSoboliev) [reply location_register/converter/drv.py:15]: Fixed

File: business_register/converter/fop.py (hunk 7 above)
Comment (IegorSoboliev) [reply business_register/converter/fop.py:12]: Done

File: business_register/converter/kved.py (hunk 6 above)
Comment (IegorSoboliev) [reply business_register/converter/kved.py:6]: Done. But, please, explain why we should put a class field to __init__?

Comment (roman-tiukh) [reply business_register/converter/kved.py]: ok, on next meeting

**Comment by roman-tiukh** [comment]
you must resolve conflicts

**Pull request merged**